networking/kvm.go

```diff
@@ -436,6 +441,15 @@ func kvmSetup(podRoot string, podID types.UUID, fps []ForwardedPort, netList com
 		},
 	}
 	var e error
+
+	// If flannel network is set as default in CNI configuration
```

Comment:
This should be independent from flannel networking (which only uses delegate to set this in primitive plugin - which defaults to be `bridge`). This should be accessible in `bridge` networking.